st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -212,8 +261,37 @@ def get_system_information():
     return system_information
 
 
+def format_output_filename(cmd):
+    """"
+    Format the file name such as removing white spaces and special characters.
+    :param cmd: shell command
+    :return: formatted output file name
+    :rtype: ``str``
+    """
+    for char in cmd:
+        if char in ' !@#$%^&*()[]{};:,./<>?\|`~=+"':
+            cmd = cmd.replace(char, "")
+    return cmd
+
+    
+def get_commands_output(config_yaml):
+    """"
+    Get output of the required shell command and redirect the output to a file.
+    :param config_yaml: config yaml file name
+    :return: output file paths
+    :rtype: ``list``
+    """
+    commands_list = get_config_details(config_yaml, 'shell_commands')
+    output_files_list = []
+    for cmd in commands_list:
+       output_file = "/tmp/%s.txt" % format_output_filename(cmd)
+       os.system("%s > '%s'" % (cmd, output_file))
```

Comment:
This should probably be using the `subprocess` module rather than `os.system`.  See https://www.python.org/dev/peps/pep-0324/